spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala

```diff
@@ -35,6 +35,10 @@ private[tensor] class DenseTensor[@specialized(Float, Double) T: ClassTag](
   var nDimension: Int)(implicit ev: TensorNumeric[T])
   extends Tensor[T] {
 
+  // pre-load MKL library. If we do not do it here,
```

Comment:
If you put here, each time new DenseTensor, the MKL.isloaded will be run.

If you put in an Tensor object, the MKL.isloaded will be run only when JVM load the class